pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -34,8 +34,10 @@ const _ = cockpit.gettext;
 // Found in /usr/share/crypto-policies/policies/
 const cryptopolicies = {
     DEFAULT: _("Recommended, secure settings for current threat models."),
+    "DEFAULT:SHA1": _("DEFAULT with SHA-1 signature verification allowed"),
     FUTURE: _("Protects from anticipated near-term future attacks at the expense of interoperability."),
     LEGACY: _("Higher interoperability at the cost of an increased attack surface."),
+    "LEGACY:AD-SUPPORT": _("LEGACY with Active Directory interoperability"),
```

Comment:
Is there some way to read the available options/subpolicies from crypto-policies?

Reply:
There are many modifiers available and you can use multiple modifiers at once to change any existing policy. You can also write your own policies and modifiers and use those in any combinations. (If someone uses something different Cockpit will show it but won;t allow setting it).

This set is what was agreed on with crypto guys as sensible options that Cockpit should offer to customers as they make sense. Anything else is dangerous/weird/too unique that someone who wants to use that they surely know enough about this topic so setting it up on cmdline shouldn't be difficult for them.